### ISSUE TITLE
Proposal: Vertical Collection 5

### ISSUE THREAD

**Issue by mixonic** [html-next/vertical-collection#400]
I propose a v5 release of vertical-collection!

* Support only Ember 3.28+
* Adopt Glimmer Components for https://github.com/html-next/vertical-collection/blob/master/addon/components/vertical-collection/component.js

What else?

**Comment by runspired** [comment]
I have thoughts. But if all we did was this list I'd be happy.

**Comment by bryanaka** [comment]
@mixonic and @runspired would love to help transition to glimmer. I see that @mixonic has a pending PR in draft #403 so just let me know if there is any assistance I can provide.

We just started making use of this in our app with lots of near real-time data, so just starting to dig into the guts 😄

**Comment by NullVoxPopuli** [comment]
@mixonic is this still something you're wanting to finish?

would be great to also have gts / v2 conversion in here as well <3

**Comment by runspired** [comment]
@NullVoxPopuli I might have us take this on soon-enough tbh

**Comment by runspired** [comment]
hit this tonight trying to make the pokedex app from @ignacemaes work with latest WarpDrive stuff (figured it would be a nice quick way to do infinite scroll on the big list)

https://github.com/IgnaceMaes/ember-polaris-pokedex/

Considering we're also using it for the same in a bunch of v2 addons at Auditboard, probably one of us should make this a priority.

**Comment by NullVoxPopuli** [comment]
Yup! V2 addon conversion should be done by the end of this week at the lates
